core/src/commonTest/kotlin/StateStringTests.kt

```diff
@@ -0,0 +1,52 @@
+package com.juul.kable
+
+import com.juul.kable.State.Connected
+import com.juul.kable.State.Connecting.Bluetooth
+import com.juul.kable.State.Connecting.Observes
+import com.juul.kable.State.Connecting.Services
+import com.juul.kable.State.Disconnected
+import com.juul.kable.State.Disconnected.Status.EncryptionTimedOut
+import com.juul.kable.State.Disconnected.Status.PeripheralDisconnected
+import com.juul.kable.State.Disconnected.Status.CentralDisconnected
+import com.juul.kable.State.Disconnected.Status.Failed
+import com.juul.kable.State.Disconnected.Status.L2CapFailure
+import com.juul.kable.State.Disconnected.Status.Timeout
+import com.juul.kable.State.Disconnected.Status.LinkManagerProtocolTimeout
+import com.juul.kable.State.Disconnected.Status.UnknownDevice
+import com.juul.kable.State.Disconnected.Status.Cancelled
+import com.juul.kable.State.Disconnected.Status.ConnectionLimitReached
+import com.juul.kable.State.Disconnected.Status.Unknown
+import com.juul.kable.State.Disconnecting
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+public class StateStringTests {
+
+    @Test
+    fun testToString() {
```

Comment:
These are tests, so `test` prefix is redundant.

```suggestion
    fun toString() {
```

Or (if `toString` is reserved):

```suggestion
    fun stringRepresentation() {
```